lingetic-nextjs-frontend/app/languages/[language]/[questionListId]/results/page.tsx

```
@@ -0,0 +1,71 @@
+interface ResultsPageParams {
+  language: string;
+  questionListId: string;
+  [key: string]: string;
+}
+
+function ResultsPageComponent() {
+  const searchParams = useSearchParams();
+  const { language, questionListId } = useParams<ResultsPageParams>();
+  const playAgainRef = useRef<HTMLAnchorElement>(null);
+  
+  const total = parseInt(searchParams.get("total") ?? "0");
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)